general/app/development/release-process.md

```diff
@@ -12,27 +12,27 @@ tags:
 | 6. | Contact the marketing team announcing the new release and highlights. | Integration Lead |
 | 7. | Add new QA tests to the "Testing cases" document. New QA tests should be labeled with [qa_test_required](https://tracker.moodle.org/issues/?jql=project%20%3D%20MOBILE%20AND%20resolution%20in%20(Unresolved%2C%20Fixed)%20AND%20labels%20%3D%20qa_test_required%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC). Remove that label once are added to the document. | Tester |
 | 8. | Run `npm audit` to ensure all the dependencies are OK and check github vulnerabilities report. | Developer |
 | 9. | Complete all TODOs related with the upcoming release, which are marked in code with a comment starting with @todo [version-number] (for example, before releasing 4.1 we'd search for comments starting with @todo [4.1]) | Developer |
-| 10. | Freeze Cordova plugins and JavaScript libraries versions (node modules) in the integration branch. | Developer |
+| 10. | Freeze Cordova plugins and JavaScript libraries versions (node modules) in the main/version branch. | Developer |
 | 11. | **Start testing** | Tester |
 
 ## The release day
 
 | **#** | **Task** | **Responsibility** |
 |---|---|---|
-| 1. | Fix the version name in `integration:config.xml`, `integration:package.json` and `integration:moodle.config.json` (remove the `-dev`). Remember to update in config.xml the value of `AppendUserAgent` as well. | Developer |
+| 1. | Fix the version name in `config.xml`, `package.json` and `moodle.config.json` (remove the `-dev`). Remember to update in config.xml the value of `AppendUserAgent` as well. | Developer |
 | 2. | Set the right version number for the new Moodle LMS major release in the site.js constant `MOODLE_RELEASES`. | Developer |
-| 3. | Integrate the [integration branch onto the master one](https://github.com/moodlehq/moodleapp/compare/master...integration). | Integration Lead |
+| 3. | Launch the internal release github workflow. | Integration Lead |
 | 4. | Do some testing with the production builds before sending the application to the stores (overall testing to see that nothing is broken because of the branch merge done in step 2): <ul><li>Correct definitive version name displayed (and commit)</li><li>AOT compiling worked</li><li>Language strings synchronised from lang.moodle.net</li><li>New language packs added</li></ul> | All the team |
 | 5. | Send the applications to the stores for review. | Integration Lead |
-| 6. | Create a TAG/Release in github ([moodlehq/moodleapp](https://github.com/moodlehq/moodleapp/releases)) with the version number. | Integration Lead |
+| 6. | Create a TAG/Release in github ([moodlehq/moodleapp](https://github.com/moodlehq/moodleapp/releases)) with the version number. | Auto |
```

Comment:
Does this "Auto" mean that this happens automatically? In that case, I think I'd just remove it from this list, given that this is supposed to be a list of things that have to be done manually.

Reply:
I'll add a check then